tensorflow/python/kernel_tests/py_func_test.py

```diff
@@ -91,7 +91,7 @@ def testLarge(self):
       x = tf.zeros([1000000], dtype=np.float32)
       y = tf.py_func(lambda x: x + 1, [x], [tf.float32])
       z = tf.py_func(lambda x: x * 2, [x], [tf.float32])
-      for _ in xrange(100):
+      for _ in range(100):
```

Comment:
For this size of range it doesn't matter much, but better to add a "from six.moves import xrange" to get xrange back.